### PR TITLE
some edge cases & merge probs fixed

### PR DESCRIPTION
**Pull request by eripheebs**

### CODE REVIEW

File: backend/models/spotifyPlaylist.js [hunk 1]
```diff
@@ -10,71 +10,75 @@ var spotifyApi = new SpotifyWebApi({
   clientSecret : clientSecret
 });
 
+var confirmationString = "Your playlist has been created!"
+var errorString1 = "Something went wrong when creating playlist!"
```
Comment (JojSh) [inline backend/models/spotifyPlaylist.js]: maybe rename playlistError & addTracksError (or something else descriptive)

File: frontend/www/js/controllers/playfestController.js [hunk 2]
```diff
@@ -22,4 +17,18 @@ playfestApp.controller('playfestController', ['TalkToBackendService', '$scope',
...
 
+  function _makeHash(playlistName, arrayOfArtists){
+    return { "playlistName" : playlistName, "arrayOfArtists" : arrayOfArtists };
+  }
+
+  function _checkIfNamed(playlistName, data){
+    if (playlistName == null){
+      $scope.confirmationData = "You must enter a playlist name";
+    } else {
+      $scope.confirmationData = null;
```
Comment (robota-x) [inline frontend/www/js/controllers/playfestController.js]: maybe put something like 'creating your playlist, please wait', to give feedback it has been actually clicked?

**Comment by JojSh** [comment]
:shipit:

**Comment by robota-x** [comment]
:shipit: beautiful job

**Comment by jazzygasper** [comment]
:shipit:

**Pull request merged**